src/commands/lambda/__tests__/instrument.test.ts

```diff
@@ -1,9 +1,9 @@
 // tslint:disable: no-string-literal
 jest.mock('fs')
+jest.mock('chalk')
```

Comment:
As it is mocked globally I do not think it is required to mock it here anymore

```suggestion
```

Reply:
Thanks!